### DISCUSSION TITLE
Image detail information source

### DISCUSSION THREAD

**Discussion by Hiroshi-Ikeda** [ImageMagick/ImageMagick#6483] [Help] [answered]
### ImageMagick version

ImageMagick 7.1.1-8 Q16-HDRI x64

### Operating system, version and so on

Windows11 64bit

### Description

Hello ImageMagick Community,

I always use Imagemagick very usefully.

Please how to read the output of "magick identify -verbose image.jpg".

The colorspace that appears in the first line of the output is described as sRGB(Colorspace: sRGB), but the colorspace described in Exif(exif:ColorSpace:) is 65535(not sRGB).

What part of the image file is referred to for the first sRGB to determine that it is sRGB?

>magick identify -verbose image.jpg

Image:
  Filename: image.jpg
  Permissions: rw-rw-rw-
  Format: JPEG (Joint Photographic Experts Group JFIF format)
  Mime type: image/jpeg
  Class: DirectClass
  Geometry: 6720x4480+0+0
  Resolution: 300x300
  Print size: 22.4x14.9333
  Units: PixelsPerInch
  Colorspace: sRGB <<<<<<<<<<<
  Type: TrueColor
  ...
 Properties:
  ...
    exif:ColorSpace: 65535  <<<<<<<<<<<
  ...

**Comment by snibgo** [comment, marked as answer]
Exif colorspace of 65535 means "uncalibrated". See https://exiftool.org/TagNames/EXIF.html .

Like most software, IM assumes that images that are not encoded in a specific colorspace are actually sRGB.

IM's verbose info "Colorspace: sRGB" can be misleading. If the image has three channels, representing R and G and B, and the color primaries are defined by an embedded ICC profile, IM's "Colorspace" will claim the image is either sRGB (or RGB), meaning the image has a non-linear (or linear) Transfer Response Curve.

**Reply by Hiroshi-Ikeda** [reply]
Thank you for your reply.

> IM's verbose info "Colorspace: sRGB" can be misleading.

I understand that 24-bit RGB images are usually treated as being in the sRGB color space.
The original image file with a properly attached metafile looks fine, but I should be wary of images that don't.
